### PR TITLE
Az Migrate Module Bug Fixes

### PR DESCRIPTION
**Pull request by prsadhu-ms-idc**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by kuchatur-ms** [comment]
@prsadhu-ms-idc I've validated the PR and my changes are there.

File: src/Migrate/custom/New-AzMigrateProject.ps1 [hunk 1]
```diff
@@ -15,59 +15,60 @@
...
+        # Specifies the project properties.
+        ${Property},
+
         [Parameter()]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Runtime.DefaultInfo(Script='(Get-AzContext).Subscription.Id')]
-        [System.String[]]
+        [System.String]
         # Specifies the subscription id.
         ${SubscriptionId}
```
Comment (isra-fel) [inline src/Migrate/custom/New-AzMigrateProject.ps1:63]: In PowerShell you can pass 1 single value to a parameter of a collection type. For example, for a `-SubscriptionId <string[]>` it's totally fine to `-SubscriptionId "one-string-instead-of-an-array"`. That's why we always model ${SubscriptionId} as an array, so please change it back

**Review by isra-fel — CHANGES_REQUESTED** [review]
Added a few suggestions, and don't forget to rebuild docs :)

File: src/Migrate/custom/New-AzMigrateProject.ps1 (hunk 1 above)
Comment (prsadhu-ms-idc) [reply src/Migrate/custom/New-AzMigrateProject.ps1:63]: Hi, In Az Migrate, we would like to operate on only one subscription ID at a time, and not on multiple, as our service has limitations. Hence this behavior.

Comment (isra-fel) [reply src/Migrate/custom/New-AzMigrateProject.ps1:63]: If you supply multiple sub ids, the cmdlet will send multiple requests, each of them to 1 sub, so service side will always receive only 1 sub id per request. 
(This is handled by generated code)

Comment (prsadhu-ms-idc) [reply src/Migrate/custom/New-AzMigrateProject.ps1:63]: But our service/product is not designed like that. So Migrate Project will use 1 Resource Group , 1 Subscription ID in it's life time. Our custom code will not know how to handle / collate information from different subscription ids, as there will be nothing common between them.

Comment (prsadhu-ms-idc) [reply src/Migrate/custom/New-AzMigrateProject.ps1:63]: Multiple sub ids, will be fundamentally multiple customers for us.

Comment (isra-fel) [reply src/Migrate/custom/New-AzMigrateProject.ps1:63]: OK got it!

File: src/Migrate/custom/New-AzMigrateProject.ps1 [hunk 2]
```diff
@@ -15,59 +15,60 @@
 
 <#
 .Synopsis
-Get All servers in a migrate project.
+Get All servers in a migrate project. New proj.
 .Description
 Get Azure migrate server commandlet fetches all servers in a migrate project.
```
[the remark this answers is not in the document]
Comment (kuchatur-ms) [reply src/Migrate/custom/New-AzMigrateProject.ps1]: fixed.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**